### ISSUE TITLE
asymptomatic elevated creatine kinase

### ISSUE THREAD

**Issue by LZ-NMD** [obophenotype/human-phenotype-ontology#11767]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
